hdk-rust/src/api/keystore.rs

```diff
@@ -24,13 +24,13 @@ pub fn keystore_derive_seed<S: Into<String>>(
     src_id: S,
     dst_id: S,
     context: S,
```

Comment:
@willemolding I notice that for `call` and `link_entries` you give the three stringish params their own generic type (I presume so that they can each be different implementations of `Into<String>`). Would this be a useful approach for the keystore functions too?

Reply:
yes! Looks like I missed this one